Survey end date will be dropped, so remove a mention of that

rgommers opened the pull request:
(after talking to @InessaPawson)

InessaPawson commented:
@rgommers I suggest the following wording in the news bar:

`title:` NumPy Community Survey is LIVE! 
`content: `Help us make NumPy better. 

And let’s link to the survey directly from the homepage. Here is a link: https://umdsurvey.umd.edu/jfe/form/SV_8bJrXjbhXf7saAl

rgommers commented:
Updated with the text, and also added your tweak of the logo news item @InessaPawson.

I'd prefer not to mess with the news bar linking, both because it's hard to do and because it's the only link to the whole news page.
